src/overlays/actors/ovl_En_Ge2/z_en_ge2.c

```diff
@@ -556,6 +556,7 @@ void EnGe2_Update(Actor* thisx, PlayState* play) {
     } else if (this->collider.base.acFlags & AC_HIT) {
         if ((this->collider.info.acHitInfo != NULL) &&
             (this->collider.info.acHitInfo->toucher.dmgFlags & DMG_HOOKSHOT)) {
+            // !@bug duration parameter is larger than 255 which messes with the internal bitpacking of the colorfilter
```

Comment:
It's probably note-worthy too that the timer (400) ends up being only 144
To be clear this has an impact on gameplay, this timer is the one used as the stun timer when hookshotting gerudos

And just a format thing but `//! @bug`, right?